docs/contributing.md

```diff
@@ -30,6 +30,22 @@ Although we greatly appreciate any and all contributions to the project, there a
 
 Please be sure to take a careful look at our [Code of Conduct](https://github.com/OWASP/owasp-mastg/blob/master/.github/CODE_OF_CONDUCT.md "Code of Conduct") for all the details and [ask us](contact.md) in case of doubt.
 
+## Can my tool be added?
+
+The goal of the MASTG is to be as accessible and up-to-date as possible. For this reason, we prioritize including tools that meet the following criteria:
+
+- Open-source
+- Free to use
+- Capable of analyzing recent Android/iOS applications
+- Regularly updated or stable by nature
+- Actively maintained over a period of time
+- Strong community support
+- Solve a unique problem
```

Comment:
This sounds like the tool solves exactly "one single" problem. I guess we mean something like "it solves certain problems in a unique or innovative manner", right? I'm just not sure how to express it.

Reply:
It wasn't necessarily what I meant, but I do like your phrasing